dev/test-fpindex-api.py

```
@@ -0,0 +1,196 @@
+
+FPINDEX_URL = "http://localhost:6081"
+INDEX_NAME = "test"
+
+def test_health():
+    """Test health endpoint"""
+    print("Testing health endpoint...")
+    try:
+        resp = requests.get(f"{FPINDEX_URL}/_health")
+        print(f"Health check: {resp.status_code} - {resp.text}")
+        return resp.status_code == 200
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)